### PR TITLE
i18n(es): translate `cards`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- What does this PR change? Give us a brief description.
Translates `cards` to spanish

### CODE REVIEW

File: docs/src/content/docs/es/components/cards.mdx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+import { Card } from '@astrojs/starlight/components';
+
+Para mostrar contenido en una caja que coincida con los estilos de Starlight, usa el componente `<Card>`.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<Card slot="preview" title="Moons" icon="moon">
```
Comment (HiDeoo) [inline docs/src/content/docs/es/components/cards.mdx:16]:
```suggestion
<Card slot="preview" title="Lunas" icon="moon">
```

We can translate the title in this preview but not sure about my suggestion.

File: docs/src/content/docs/es/components/cards.mdx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+### Agregar iconos a las tarjetas
+
+Incluye un icono en una tarjeta usando el atributo [`icon`](#icon) establecido en el nombre de [uno de los iconos integrados de Starlight](/es/reference/icons/#todos-los-iconos).
+
+<Preview>
+
+```mdx 'icon="star"'
+import { Card } from '@astrojs/starlight/components';
+
+<Card title="Stars" icon="star">
```
Comment (HiDeoo) [inline docs/src/content/docs/es/components/cards.mdx:65]:
```suggestion
<Card title="Estrellas" icon="star">
```

Same here

File: docs/src/content/docs/es/components/cards.mdx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+	Sirius, Vega, Betelgeuse
+</Card>
+```
+
+<Fragment slot="markdoc">
+
+```markdoc 'icon="star"'
+{% card title="Stars" icon="star" %}
```
Comment (HiDeoo) [inline docs/src/content/docs/es/components/cards.mdx:73]:
```suggestion
{% card title="Estrellas" icon="star" %}
```

File: docs/src/content/docs/es/components/cards.mdx [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+Sirius, Vega, Betelgeuse
+{% /card %}
+```
+
+</Fragment>
+
+<Card slot="preview" title="Stars" icon="star">
```
Comment (HiDeoo) [inline docs/src/content/docs/es/components/cards.mdx:80]:
```suggestion
<Card slot="preview" title="Estrellas" icon="star">
```

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). I left a few suggestions to translate the example titles but other than that, everything looks good to me.

**Comment by dreyfus92** [comment]
Thanks @HiDeoo for the review 🫡

**Review by HiDeoo — APPROVED** [review]
Thanks for the updating the PR, looks great to me now :star2:

**Pull request merged**